### ISSUE TITLE
Change project structure

### ISSUE THREAD

**Issue by kirilldakhniuk** [TeriyakiBomb/nova-artisan#2]
Hey,
I'm really interested in ability to change the default project structure, for example I use modules structure and I want to create a Model under modules/{module}/Models, etc. Like the Laravel Idea plugin does for PHPStorm. Is it something you can do?

**Comment by TeriyakiBomb** [comment]
I'd recommend having a look at the extension (it's actually pretty simple) and making some tweaks. If I get some more requests for this I could certainly add it to the extension down the line. There could potentially be an area in the extension config to determine explicit paths where the tasks point? Ultimately this is just running artisan commands under the hood.

**Comment by kirilldakhniuk** [comment]
Yeah, I guess it's possible to use a settings for the extension. I'll look into it when I have a chance. Thanks.
